src/Raven.Client/Documents/Indexes/ExpressionStringBuilder.cs

```diff
@@ -841,6 +841,43 @@ protected override Expression VisitConstant(ConstantExpression node)
                 Out('M');
                 return node;
             }
+            if (node.Value is DateTime dateTime && dateTime.Equals(default))
+            {
+                Out("default(");
+                Out(typeof(DateTime).ToString());
```

Comment:
nameof would be enough? Indexes do have `System` using IIRC?

Reply:
Changed